libraries/SPI/src/SPI.h

```diff
@@ -55,6 +44,17 @@ enum SPITransferMode {
   SPI_LAST
 };
 
+// Indicates the user controls himself the CS pin
+#define CS_PIN_CONTROLLED_BY_USER  NUM_DIGITAL_PINS
+
+// Defines a default timeout delay in seconds for the SPI transfer
+#define SPI_TRANSFER_TIMEOUT    5000
+
```

Comment:
This seems not related to CS related change and should be in a different commit.
The timeout was hardcoded to 10000 before and now is defined as 5000. 
Is it really 5000 seconds ??   Do we consider that timeout can be fixed, whatever the number of bytes and frequency ?

Reply:
It is not 5000 seconds but 5000 milliseconds. I took advantage of the fix to standardize the timeout value. 
This timeout doesn't change whatever the number of bytes or frequency.

Reply:
The comment above the definition is
// Defines a default timeout delay in seconds for the SPI transfer
so you need to change that ... 
How many bytes does it allow to send @ slowest supported frequency ?

Reply:
Unless I am mistaken, we can send 78kbytes at slowest frequency = 125000.

Reply:
ok that sounds like a reasonable number. Even 1 sec might be enough then (~15kBytes) ...